server/server.go

```diff
@@ -350,6 +499,8 @@ func New(cfg *config.Config) (http.Handler, error) {
 
 	m.Add("GET", "/api/v1/search/:backend", srv.Handler(srv.ServeAPISearch))
 	m.Add("GET", "/api/v1/search/", srv.Handler(srv.ServeAPISearch))
+	m.Add("GET", "/api/v1/langserver/jumptodef", srv.Handler(srv.ServeJumpToDef))
```

Comment:
I don't love `langserver` as the path component here. I'd like either `lsp`, which is the standard abbreviation for the language server protocol, or perhaps something like `sema` or `analyze` which gets at the *logical* function of these endpoints -- these are the endpoints that provide semantic source-code analysis. It happens to be backed by LSP today, but in principle you could imagine swapping out the server implementation for something else, while preserving at least approximately the same server API…

How do you feel about using `lsp` instead of `langserver` here? There's a perfectly good standard acronym for what we're using

Reply:
I agree that`lsp` is better. Fixed 😄